src/middleware/error_handling.ts

```diff
@@ -70,11 +79,15 @@ export function errorHandler(
     if (res.headersSent) {
         return next(err);
     }
-    if ((err as any).isAPIError || (err as any).statusCode) {
-        const { statusCode, errorBody } = generateError(err);
-        res.status(statusCode).send(errorBody);
-        return;
-    } else {
-        return next(err);
+
+    const { statusCode, errorBody } = generateError(err);
+    res.status(statusCode).send(errorBody);
+
+    // If the error is an internal error, log it with the stack!
+    // All other error responses are logged as part of request logging
+    if (statusCode === HttpStatus.INTERNAL_SERVER_ERROR) {
+        // hack (xianny): typeorm errors contain the SQL query which breaks the docker char limit and subsequently breaks log parsing
+        logger.error({ ...err, query: undefined});
```

Comment:
Right now it seems like this is the only field that breaks log parsing. If something else breaks it, let's look into truncating JSON objs at that time. Feels like it will take too long to be worth it for now, unless anyone has an easy suggestion to implement?

Reply:
This is fine. Might be good to link the actual issue which I believe is a `fluentd` issue.